api/Performance.json

```
@@ -29,12 +29,12 @@
           },
           "nodejs": [
             {
-              "version_added": "16.0.0"
+              "version_added": "18.8.0"
             },
             {
               "version_added": "8.5.0",
               "partial_implementation": true,
-              "notes": "Exported from the <code>perf_hooks</code> module"
+              "notes": "Only available as an internal instance."
```

How could users interact with this interface, if it wasn't exported prior to 18.8.0?

---

though the interface can't be accessed directly, but it can be accessed via `performance` property: `require('perf_hooks').performance.__proto__.constructor`

---

> it can be accessed via `performance` property: `require('perf_hooks').performance.__proto__.constructor`

Should we mention this explicitly? I feel like the previous note was somewhat more informative.

```suggestion
              "notes": "Only available as an internal instance (`require('perf_hooks').performance.__proto__.constructor`)."
```

---

@skyclouds2001 What do you think about this suggestion? 👆

---

I think the info of how to access the interface is not required, as in most case user will not access this interface directly but use its properites and methods; and since the info is only for old node.js version, very fewer user will use it.